### ISSUE TITLE
[Feature request] Button to copy SHA to clipboard

### ISSUE THREAD

**Issue by gobater** [ForkIssues/TrackerWin#823]
Hi,

I do a mixed use of git. I perform most of the tasks from the GUI (Fork) but sometimes I do some other tasks from a shell. For that, I navigate in Fork throught the history of my project, I copy the SHA of some interesting commit into the clipboard and then I continue from the command line.

It would be helpful (also for others) to have a button (next to the SHA) to copy it to the clipboard, similar to GitHub.

Regards

**Comment by Otiel** [comment]
There's already a context menu to copy the commit SHA. I think a button next to each commit would be too much for the interface.

Maybe a new keyboard shortcut (<kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>C</kbd>) would be enough?

**Comment by gobater** [comment]
I was not pretending to put the button in the log. I meant a button next to the SHA (in the commit description). See attached screenshot.

I do not know which use case is behind the current "Ctrl + C" which is copying the short SHA + subject of the commit, but "Ctrl + Shift + C" for the SHA does not sound very intuitive to me... but it could be an option. However, a button in the description would be extremely helpful for people not able to remember shortcuts

![image](https://user-images.githubusercontent.com/23657567/84063948-218f2400-a9c2-11ea-9f92-f3b1ee691e74.png)

**Comment by Otiel** [comment]
My bad, that's a good idea indeed 👍

**Comment by gobater** [comment]
Hi @DanPristupov ,

Any plans to implement this feature? Even if not highest prio, it does not seem a big deal to me

**Comment by DanPristupov** [comment]
This controls is simple. But below is the parents field which can contain multiple SHAs for merge commit.
We also have abbreviated SHAs in the header in Changes tab and when compare a pair of commits.

It's not as simple as it sounds. The change must be consistent. A similar feature has existed in our todo list since the beginning, but I don't have a clear vision how it should look like.

**Comment by gobater** [comment]
Agree... if you want to add the "icon" also to the parent, for merge commits (with multiple parents) it might look weird...

Happy to know that a "similar feature" is in your todo list... thanks for keeping us updated!
